core-feature-pack/src/main/resources/content/bin/elytron-tool.ps1

```diff
@@ -9,8 +9,11 @@ $scripts = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.FullName;
 $SCRIPT_NAME = $MyInvocation.MyCommand | select -ExpandProperty Name
 $SCRIPT_NAME = "{" + $SCRIPT_NAME + "}"
 
-$ELYTRON_TOOL_OPTS = Process-Script-Parameters -Params $ARGS
-$ELYTRON_TOOL_OPTS = $SCRIPT_NAME + $ELYTRON_TOOL_OPTS 
+$ELYTRON_TOOL_OPTS=@()
+if ($ARGS.Count -gt 0){
+  $ELYTRON_TOOL_OPTS+=$SCRIPT_NAME + $ARGS[0]
+  $ELYTRON_TOOL_OPTS+=$ARGS[1..$ARGS.Count]
```

Comment:
@ctomc actually the first parameter should contain "{script_name}arg0" and rest of parameters should be the same as they are passed to the script. It works like this across scripts (.bat, .sh) and java code will cope with it.
The reason why is that we can have couple of scripts (.sh, .bat, .sh1) to start the tool and also java -jar as the last possibility.